src/net.cpp

```diff
@@ -1777,20 +1773,6 @@ void ThreadOpenConnections()
                     nDisconnects++;
                 }
             }
-#ifdef BITCOIN_CASH
-            // Disconnect a node that is not BitcoinCash capable if all outbound slots are full and we
-            // have not yet connected to enough BitcoinCash nodes.
-            else if (nOutbound >= nMaxOutConnections && nBitcoinCash <= min(nMinBitcoinCashNodes, nMaxOutConnections) &&
-                     nDisconnects < MAX_DISCONNECTS && IsChainNearlySyncd())
-            {
-                if (ptemp != nullptr)
-                {
-                    ptemp->fDisconnect = true;
-                    fDisconnected = true;
-                    nDisconnects++;
-                }
-            }
-#endif
```

Comment:
This should be removed , yes, but we should still present the service bit in case older nodes or other services still are expecting it.

Reply:
I would recommend making this a separate PR and merge right away for the bug fix release...this disconnecting of nodes doesn't need to be there anymore...